rasa/core/brokers/kafka.py

```diff
@@ -70,6 +73,9 @@ def __init__(
         self.ssl_keyfile = ssl_keyfile
         self.ssl_check_hostname = ssl_check_hostname
 
+        if partition_key:
+            self.partition_key = bytes(partition_key, encoding="utf-8")
```

Comment:
how about using the existing constant for this?
```suggestion
            self.partition_key = bytes(partition_key, encoding=DEFAULT_ENCODING)
```